custom/icds_reports/tasks.py

```diff
@@ -1219,7 +1219,7 @@ def build_incentive_report(agg_date=None):
                 build_incentive_files.delay(location, agg_date, file_format, 3, location.parent.parent, location.parent)
 
 
-@task(queue='icds_dashboard_reports_queue')
+@task(queue='icds_dashboard_reports_queue', serializer='pickle')
```

Comment:
Except for month, I think this could be fixed by changing the parameters to:

location -> location_id
state -> state_name
district -> district_name

add block_name=None

Do you happen to know how we plan to handle dates later? Could probably add some date parsing as the first line?